monai/data/image_reader.py

```diff
@@ -810,49 +808,57 @@ def _check_level(self, img, level):
         return level
 
     def _get_image_size(self, img, size, level, location):
+        """
+        Calculate the maximum region size for the given level and starting location (if size is None).
+        Note that region size in OpenSlide and cuCIM are WxH (but the final image output would be HxW)
+        """
+        if size is not None:
+            return size[::-1]
```

Comment:
so this could be simplified? https://github.com/Project-MONAI/MONAI/blob/f8e5994d1d4ac73a3bc79270bf6a6f1eba5d4f2c/monai/data/image_reader.py#L856-L859

Reply:
Good catch! Thanks